packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -40,6 +42,31 @@ export const EventTypeDescription = ({
 
   const paymentAppData = getPaymentAppData(eventType);
 
+  const showManagedBadge = eventType.metadata?.managedEventConfig && !isPublic;
+  const numOfManagedByAdmins = eventType.managedBy?.admins.length || 0;
+  const showManagedByToolTip = showManagedBadge && numOfManagedByAdmins > 0;
```

Comment:
We don't need the `showManagedByToolTip` variable

Reply:
i thought since this condition is used at 2 places, we can have this flag, ya since we are going with single batch (this was my initial approach) removed this.